lib/include/sdotypes.h

```diff
@@ -115,6 +115,7 @@ bool sdo_string_read(sdor_t *sdor, sdo_string_t *b);
 #define SDO_GID_BYTES (128 / 8)
 #define SDO_NONCE_BYTES (128 / 8)
 #define SDO_NONCE_FIELD_BYTES 32
+#define SDO_UEID_BYTES (128 / 8) + 1
```

Comment:
the "+ 1" looks troublesome - can you add a comment to explain it?

Reply:
Updated.